tests/dummy/src/02_advanced/README.md

```diff
@@ -0,0 +1,5 @@
+# README.md tests
+
+## image link
+
+![Awesome image in the same folder](Epub_logo.svg "Awesome")
```

Comment:
Let's add following image link to that page.

```
![Awesome image in the same folder](Epub_logo.svg "Awesome")

![Awesome image 2 in the another folder](..\\reddit.svg)

![Awesome image 3 in the another folder](..\\assets\\rust-logo.png)

```

I've found that page renders correctly on Linux, but wrong on Win11. Images are not shown due to incorrect paths on Win. Can you check and confirm that ?

Reply:
I've replalced all / by \\ in all pages for testing on Win.

Reply:
That can be related to issue - https://github.com/Michael-F-Bryan/mdbook-epub/issues/126 but we need support for both cases.